app/integration/project-generator/src/main/resources/io/syndesis/integration/project/generator/templates/redhat/application.properties.mustache

```diff
@@ -11,3 +11,6 @@ management.port = 8081
 # disable all management enpoints except health
 endpoints.enabled = false
 endpoints.health.enabled = true
+
+# disable default health indicators, especially jms
+management.health.defaults.enabled = false
```

Comment:
just out of curiosity, why is this required ?

Reply:
When Spring Boot sees ActiveMQ libraries, it thinks it helping by injecting a `vm://localhost` connection factory in JMS health indicator and creating an in vm broker to monitor it. 

Which fails, since the vm transport is part of the ActiveMQ Broker libraries, not client. So it spectacularly barfs, over and over in the logs.

Reply:
On the question about the need to lookup the pod, I agree. We should figure out a way to make it easier to do that from the UI.

Reply:
With only `integrationId` the method `getIntegrationMetricsSummary` is only useful for summary metrics, not per deployment metrics. We should refactor `MetricsProvider` methods to:
```
getDeploymentMetrics ( integrationId, deploymentId )
getIntegrationMetrics ( integrationId )
```
Where the latter method aggregates metrics for counters, and uses appropriate min/max value for timestamps, etc.